From b0104476f8f6b20bad18ae8be23530ecac01f3fe Mon Sep 17 00:00:00 2001
From: Joseph Chen <chenjh@rock-chips.com>
Date: Sun, 11 Feb 2018 16:13:11 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3326/px30: add pmic pwrkey node

pwrkey node is necessary for U-Boot RK817/RK809 pwrkey driver,
because the device-driver model requires it to make a match.

Change-Id: Ica766232646ba93508592ab1a20b65157859d688
Signed-off-by: Joseph Chen <chenjh@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts  | 4 ++++
 arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts  | 4 ++++
 arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts | 4 ++++
 arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts | 4 ++++
 4 files changed, 16 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
index c12b5bca62ef..e50293c7dbfa 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
@@ -332,6 +332,10 @@
 		vcc8-supply = <&vcc3v3_sys>;
 		vcc9-supply = <&vcc5v0_sys>;
 
+		pwrkey {
+			status = "okay";
+		};
+
 		regulators {
 			vdd_logic: DCDC_REG1 {
 				regulator-always-on;
diff --git a/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts b/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
index b9f2c9e30108..dd855d95b450 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
@@ -332,6 +332,10 @@
 		vcc8-supply = <&vcc3v3_sys>;
 		vcc9-supply = <&vcc5v0_sys>;
 
+		pwrkey {
+			status = "okay";
+		};
+
 		regulators {
 			vdd_logic: DCDC_REG1 {
 				regulator-always-on;
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
index ebbf0e3ebd6d..fd94f077ac56 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
@@ -268,6 +268,10 @@
 		vcc8-supply = <&vccsys>;
 		vcc9-supply = <&dcdc_boost>;
 
+		pwrkey {
+			status = "okay";
+		};
+
 		regulators {
 			vdd_logic: DCDC_REG1 {
 				regulator-always-on;
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
index 38840d7c4d75..b7baa49c7ae8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
@@ -362,6 +362,10 @@
 		vcc8-supply = <&vccsys>;
 		vcc9-supply = <&dcdc_boost>;
 
+		pwrkey {
+			status = "okay";
+		};
+
 		regulators {
 			vdd_logic: DCDC_REG1 {
 				regulator-always-on;
-- 
2.35.3

